addons/skin.estuary/1080i/DialogVolumeBar.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <window>
-    <zorder>4</zorder>
```

Comment:
please keep this one, it is needed to show volume on top of darkened backgrounds of dialogs.